autoprotocol/protocol.py

```diff
@@ -5081,6 +5082,27 @@ def autopick(
             Source wells are not all from the same container
 
         """
+        groups = []
```

Comment:
It may be good to have a check that checks that the group is an instance of the AutopickGroup class.